main.go

```diff
@@ -100,6 +104,12 @@ func main() {
 	if *debug {
 		log.SetLevel(log.DebugLevel)
 	}
+
+	if *enterPwd {
+		fmt.Println("Enter registry login password:")
+		fmt.Scanln(password)
```

Comment:
This is not the proper way to read passwords in terminal. See [here](https://godoc.org/golang.org/x/crypto/ssh/terminal#ReadPassword)